include/deal.II/base/array_view.h

```diff
@@ -1134,6 +1134,21 @@ make_array_view(const Table<2, ElementType> &                   table,
 
 
 
+/*
+ * Create a view that doesn't allow the container it points to to be modified.
+ * This is useful if the object passed in is not `const` already.
```

Comment:
since I got confused what this function does, would you be willing to state in the documentation that this function returns an object of type `ArrayView<const T>` where `T` is the element type of the container?

Reply:
Yes, I improved that while squashing.